project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/templates/c4m_document_widget.tpl.php

```diff
@@ -6,22 +6,24 @@
  */
 ?>
 <div class="form-group input-wrapper file-wrapper" id="add-document-form">
-  <label class="control-label"><?php print $title; ?></label>
```

Comment:
Yup, I've [inlined](https://github.com/capacity4dev/capacity4more/pull/1119/commits/07cffc4a1023b7260d6c689c8949b7054f7f4348) required's `*` (asterisk) and [corrected](https://github.com/capacity4dev/capacity4more/pull/1119/commits/5198ee306c02445a89b255cbc52c4baa26aac27d) the indention.